### PR TITLE
Packaging/marketplace

### PR DESCRIPTION
**Pull request by siddhantk232**

### CODE REVIEW

File: src/apps/inventory/components/DataCard/index.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function DataCard({ title, quantity, actionText }) {
       >
          <Text as="title">{title}</Text>
 
-         <Text as="h2">{quantity}</Text>
+         <Text as="h2">{quantity === 0 ? quantity : 'N/A'}</Text>
```
Comment (prvnbist) [inline src/apps/inventory/components/DataCard/index.jsx:17]:
```suggestion
         <Text as="h2">{quantity || 'N/A'}</Text>
```

File: src/apps/inventory/packagingHub/components/Cart/PaymentTunnel.jsx [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+      error,
+      data: { available: availableBalance = [] } = {},
+   } = useOrganizationBalanceInfo(org[0]?.organization?.stripeAccountId)
+
+   if (error) {
+      console.log(error)
+      return toast.error(error.message)
+   }
+
+   if (loading || balanceLoading || transactionLoading) return <Loader />
```
Comment (prvnbist) [inline src/apps/inventory/packagingHub/components/Cart/PaymentTunnel.jsx:210]:
```suggestion
   if ([loading, balanceLoading, transactionLoading].some(node => node)) return <Loader />
```

File: src/apps/inventory/components/DataCard/index.jsx (hunk 1 above)
Comment (siddhantk232) [reply src/apps/inventory/components/DataCard/index.jsx:17]: in this case if the quantity is 0 which is falsy, 'N/A' will be rendered. This is not the expected behaviour here.

File: src/apps/inventory/packagingHub/components/Cart/PaymentTunnel.jsx (hunk 2 above)
Comment (siddhantk232) [reply src/apps/inventory/packagingHub/components/Cart/PaymentTunnel.jsx:210]: This looks neat but for only three variables, is using Array a good idea?

File: src/apps/inventory/packagingHub/hooks/useOrganizationBalance.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+export default function useOrganizationBalance(accountId) {
+   const [data, setData] = useState()
+   const [loading, setLoading] = useState(true)
+   const [error, setError] = useState()
+
+   useEffect(() => {
+      const abortController = new window.AbortController()
+
+      if (accountId) {
+         const url = `${PAYMENTS_URL}/${
```
Comment (prvnbist) [inline src/apps/inventory/packagingHub/hooks/useOrganizationBalance.js:14]: Couldnt you have just set the DAILYKEY_URL to have the endpoint `/payments-test` in your test environment and then `/payments` in production, why hardcode it?

Comment (siddhantk232) [reply src/apps/inventory/packagingHub/hooks/useOrganizationBalance.js:14]: I thought that the `DAILYKEY_URL` might be for other things also. That is why I am attaching `/payments` or `/payments-test` here in this hook.

Comment (prvnbist) [reply src/apps/inventory/packagingHub/hooks/useOrganizationBalance.js:14]: didnt you added that key?

Comment (prvnbist) [reply src/apps/inventory/packagingHub/hooks/useOrganizationBalance.js:14]: these endpoints are can be changed, so we cant really hard code them

Comment (siddhantk232) [reply src/apps/inventory/packagingHub/hooks/useOrganizationBalance.js]: fixed in the new commit.

**Pull request merged**